automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/cluster/ClusterPatchSpec.scala

```diff
@@ -1,57 +0,0 @@
-package org.broadinstitute.dsde.workbench.leonardo
-package cluster
-
-import org.broadinstitute.dsde.workbench.google2.MachineTypeName
-import org.scalatest.time.{Minutes, Seconds, Span}
-import org.scalatest.DoNotDiscover
-import org.broadinstitute.dsde.workbench.service.util.Tags
-
-@DoNotDiscover
-class ClusterPatchSpec extends ClusterFixtureSpec with LeonardoTestUtils {
```

Comment:
Is there another automation test where we test runtime `PATCH` functionality?

Reply:
I was thinking there's `RuntimePatchSpec`, but I guess that's for gce only, so we should keep this one, I'll revert this

Reply:
Yeah ok to remove the old PATCH route -- just want to make sure PATCH is covered in the new routes

Reply:
I'm gonna leave this spec removed but add another unit test for dataproc patch